.github/workflows/deploy.yml

```diff
@@ -0,0 +1,121 @@
+name: deploy
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - lab
+      - master
+      - published
+
+# these permissions are needed to interact with GitHub's OIDC Token endpoint.
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      -
+        name: Prepare
+        run: |
+          JEKYLL_ENV=development
+          DOCS_AWS_REGION=us-east-1
+          if [ "${{ github.ref }}" = "refs/heads/master" ]; then
+            DOCS_URL="https://docs-stage2.docker.com"
+            DOCS_AWS_IAM_ROLE="arn:aws:iam::710015040892:role/stage-docs-docker.github.io-20220816140248629900000003"
+            DOCS_S3_BUCKET="stage-docs-docker.github.io"
```

Comment:
Probably too late now (?) to rename these buckets but we should really try to get rid of that `docker.github.io` everywhere, and use `docs.docker.com` (also rename the repository back to `docs` or `documentation` or `docs.docker.com` at some point)

Reply:
I guess it's to match the name of the repository. Am I right @VictorBersy?

Reply:
Yes, it was to match the repo name, but I don't mind renaming them. It's not too late for that.
I can name everything as docs.docker.com. Ideally, it would match the future name of the repository. Let me know 🙇

Reply:
I don't think we have a final "verdict" on the repository name (but anything else than `docker.github.io` 😂)

`docker/docs` would work for me (with the caveat that we used to have a repository with that name that's now archived; https://github.com/docker-archive/docs.docker.com

Oh, LOL, and which apparently also was renamed to `docs.docker.com` 😂  so either way we would break a redirect (not really important, that stuff is pre-historic).

Reply:
Let me know when you have one, ideally before the production release, as it would make the migration more tedious 😊

Reply:
@VictorBersy if it's ok with you that _potentially_ the name of the bucket doesn't match the name of the repository, I'm inclined to name the buckets after the domain they're for (`docs.docker.com` / `docs-stage.docker.com` etc) as those are unlikely to change.

@crazy-max WDYT?

Reply:
Oh absolutely, no problem with that!